src/types.ts

```diff
@@ -42,9 +42,10 @@ export interface QueryOpts<TGraphQLVariables = OperationVariables> {
   fetchPolicy?: FetchPolicy;
   errorPolicy?: ErrorPolicy;
   pollInterval?: number;
-  client?: ApolloClient<any>;
+  client?: ApolloClient<Object>;

```

Comment:
I don't believe `ApolloClient<Object>` to be correct.  What is the rationale for changing it?

Reply:
As far as `QueryProps` interface defines `client` as [`ApolloClient<Object>`](https://github.com/apollographql/react-apollo/blob/master/src/Query.tsx#L95) I preserved it within my changes. I agree that `any` would be a better option for both interfaces.

Reply:
In this file you have changed it so I would change it back.  I'm pretty certain `Object` is incorrect.

Reply:
Reverted back `any` type.